docs/install-upgrade/config.md

```diff
@@ -138,7 +138,13 @@ Alloy](https://grafana.com/docs/alloy/latest/) on all switches to forward metric
 [Prometheus Remote-Write
 API](https://prometheus.io/docs/specs/prw/remote_write_spec/) and Loki API. Metrics includes port speeds, counters, 
 errors, operational status, transceivers, fans, power supplies, temperature
-sensors, BGP neighbors, LLDP neighbors, and more. Logs include Hedgehog agent logs.
+sensors, BGP neighbors, LLDP neighbors, and more. Logs include Hedgehog agent
+logs. Modify the URL as needed, instead of `/api/v1/push` it could be
+`/api/v1/write`; check the documentation for the data provider.
```

Comment:
Is it a "data provider"? Can't think about good name for it unfortunately...

Reply:
I also struggled, I am happy to open discussion on a better name.